### PR TITLE
Remove static headerfiles

### PR DESCRIPTION
**Pull request by astoycos**
Remove static headerfiles and instead rely on 

UAPI from https://github.com/libbpf/libbpf/tree/master/include/uapi/linux
bpf helpers from https://github.com/libbpf/libbpf/tree/master/src

from libbpf version v0.8.0 

Add new make target which handles downloading these headers if needed

### CODE REVIEW

**Comment by astoycos** [comment]
/hold

I want to get some more opinions before merging this

File: backends/ebpf/bpf/cgroup_connect4.c
Comment (dave-tucker) [inline backends/ebpf/bpf/cgroup_connect4.c:4]: What we use in `bpfd` is this:

https://github.com/redhat-et/bpfd/blob/8d837c1acb9a62018d904d1240e37376ce2e1e73/xtask/src/build_ebpf.rs#L103

Effectively, from the `libbpf` checkout `make INCLUDEDIR="./include/bpf" install_headers`.
If you invoke clang with `-I/path/to/libbpf/include' your includes then become:
```
#include <uapi/linux/bpf.h>
#include <bpf/bpf_helpers.h>
```
... which sits better with me.

Comment (astoycos) [reply backends/ebpf/bpf/cgroup_connect4.c]: ACK thanks for the review, done

**Review by dougsland — APPROVED** [review]

**Pull request merged**
